zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/repo/NotebookRepoSync.java

```diff
@@ -83,6 +84,27 @@ public void init(ZeppelinConfiguration conf) throws IOException {
       defaultNotebookRepo.init(conf);
       repos.add(defaultNotebookRepo);
     }
+
+    // convert old note file (noteId/note.json) to new note file (note_name_note_id.zpln)
+    boolean convertToNew = conf.getBoolean(ConfVars.ZEPPELIN_NOTEBOOK_NEW_FORMAT_CONVERT);
+    boolean deleteOld = conf.getBoolean(ConfVars.ZEPPELIN_NOTEBOOK_NEW_FORMAT_DELETE_OLD);
+    if (convertToNew) {
+      NotebookRepo newNotebookRepo = repos.get(0);
+      OldNotebookRepo oldNotebookRepo =
+          PluginManager.get().loadOldNotebookRepo(newNotebookRepo.getClass().getCanonicalName());
+      oldNotebookRepo.init(conf);
+      List<OldNoteInfo> oldNotesInfo = oldNotebookRepo.list(AuthenticationInfo.ANONYMOUS);
```

Comment:
would accessing as anonymous here causes some old note to be skipped?

Reply:
No, anonymous can access all notes.